GoogleDataTransport/GDTCORLibrary/GDTCORStoredEvent.m

```diff
@@ -78,8 +78,7 @@ - (nullable instancetype)initWithCoder:(nonnull NSCoder *)aDecoder {
     NSNumber *qosTier = [aDecoder decodeObjectOfClass:[NSNumber class] forKey:kQosTierKey];
     _qosTier = [qosTier intValue];
     _clockSnapshot = [aDecoder decodeObjectOfClass:[GDTCORClock class] forKey:kClockSnapshotKey];
-    _customPrioritizationParams = [aDecoder decodeObjectOfClass:[NSDictionary class]
-                                                         forKey:kCustomPrioritizationParamsKey];
+    _customBytes = [aDecoder decodeObjectOfClass:[NSDictionary class] forKey:kCustomBytesParamsKey];
```

Comment:
nit: Looks like we are still rely on `NSCoding` to encode/decode `customBytes`. Is it going to be replaced by JSON? If yes, maybe add JSON to the var name to be more explicit?

Reply:
Yeah, NSCoding will go away with the sqlite refactor. Using NSCoding for the bytes is fine because it's already bytes, whereas encoding a dictionary is worse because it will crash if the dictionary contains a value that doesn't implement the NSSecureCoding protocol.